dynamiq/nodes/tools/python.py

```diff
@@ -42,6 +43,11 @@ def restricted_import(name, globals=None, locals=None, fromlist=(), level=0):
     return importlib.import_module(name)
 
 
+class PythonInputSchema(BaseModel):
+    class Config:
+        extra = "allow"
```

Comment:
Please use pudantic V2 suggested model config
```suggestion
    model_config = ConfigDict(extra="allow")
```